src/cli_health_gateway/cli_health_gateway.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Command } from 'commander';
+import { kibanaPackageJson } from '@kbn/utils';
+import { bootstrap } from '@kbn/health-gateway-server';
+
+const program = new Command('bin/kibana-health-gateway');
+
+program
+  .version(kibanaPackageJson.version)
+  .description(
+    'This command starts up a health gateway server that can be ' +
+      'configured to send requests to multiple Kibana instances'
+  )
+  .option('-c, --config', 'Path to a gateway.yml configuration file')
+  .action(async () => {
+    return await bootstrap();
+  });
```

Comment:
Looking at the `serve` command, the arguments are passed down to the `action` handler

https://github.com/elastic/kibana/blob/90033537295b7e63b98b9f2b60eb1f321911935d/src/cli/serve/serve.js#L201-L203

So isn't the `-c` / `--config` ignored here?

Reply:
Yes the `option` doesn't serve a functional purpose here, it's only there so it shows up if someone does `./bin/kibana-health-gateway --help`